### ISSUE TITLE
Not Vuejs compatible if looking to customize. 

### ISSUE THREAD

**Issue by timothymarois** [trevoreyre/autocomplete#125]
After spending most of my day working with this, I've come to the conclusion that this was not designed for reactive frameworks. Its working too much with the DOM and preventing many things from working properly. 

You can however, use it out of the box and appears to function ok, but if you want to change the template (default slot), using your own text component and list components, you will be very disappointed. Value isnt reactive, input listeners dont work well with vuejs, while I 'rigged' it to be functional, I reached one more problem in the list elements and was not able to use my own @click for list items. 

If you try to build your own template, it just does not work properly. Until V3 is ready, this is not VueJS compatible.

**Comment by trevoreyre** [comment]
Yep, there's definitely some things to iron out for the customized templates in Vue. If you have some specific errors you're seeing, or some reproduction steps, I can try to address them when I eventually make this compatible with Vue v3.

Related: #123
